.github/workflows/morongo_integration.yml

```diff
@@ -0,0 +1,38 @@
+name: Morango Integration Tests
+
+on:
+    schedule:
+    - cron: '0 0 * * 0'
+
+jobs:
+  morango_integration_tests:
+    name: Morango Integration Tests
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.9'

```

Comment:
It would be nice to run a full matrix of supported Python versions here, as we do in the tox.yml for Sqlite backed tests.

Would need to make sure this script was invokdd before running pytest though: https://github.com/learningequality/kolibri/blob/develop/tox.ini#L28

Also worth replicating the dependency caching that is happening there to speed up execution of this test sequence if we do a matrix of Python versions.

Reply:
@rtibbles To my knowledge, GitHub Actions handles dependency caching automatically; I could be mistaken

Reply:
This is not the case, this is why Github have a dedicated action for caching https://github.com/actions/cache with instructions for different package managers.

Reply:
My mistake; I assumed it occurred automatically. I've learned something new and incorporated it